src/components/ImportForm/utils/useValidApplicationName.ts

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { useK8sWatchResource } from '@openshift/dynamic-plugin-sdk-utils';
+import { ApplicationGroupVersionKind } from '../../../models';
+import { ApplicationKind } from '../../../types';
+import { useNamespace } from '../../../utils/namespace-context-utils';
+
+const BASE_NAME = 'my-first-app';
```

Comment:
This will always result in `my-first-app-${some number}` even if I have multiple apps already created. Would be better to use generic name like `my-app` or something. Or if we want to be better at recommending name then we can say `my-first-app`, `my-second-app`, etc based on the number of applications already present.

Reply:
I think we can go ahead with simple implementation of `my-app-1`, `my-app-2`, etc for MVP.

Reply:
Makes sense, updated.